.github/workflows/main.yml

```diff
@@ -118,12 +127,16 @@ jobs:
       - name: Build and Test
         shell: bash
         run: |
-          apt-get update
-          apt-get -yq install gcc-arm-linux-gnueabihf g++-arm-linux-gnueabihf gcc-aarch64-linux-gnu
```

Comment:
These are pre-installed and routinely updated by our managed runner image, ziti-builder, which is based on Ubuntu Focal 20.04.